packages/ced/glut-link.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 504edf4..d7cb4c4 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -62,7 +62,9 @@ INSTALL_SHARED_LIBRARY( CED DESTINATION lib )
+ 
+ # link CED client library to the math library
+ FIND_LIBRARY( c_math_lib NAMES m )
+-TARGET_LINK_LIBRARIES( CED ${c_math_lib} )
++FIND_LIBRARY( glut_lib NAMES glut )
++TARGET_LINK_LIBRARIES( CED ${glut_lib} ${c_math_lib} )
```

Comment:
Could we use the glut library that we already find above?
```suggestion
+TARGET_LINK_LIBRARIES( CED ${GLUT_glut_LIBRARY} ${c_math_lib} )
```

or maybe even
```suggestion
+TARGET_LINK_LIBRARIES( CED ${GLUT_LIBRARIES} ${c_math_lib} )
```

and then not having to find the library again?